src/util/common.ts

```diff
@@ -11,18 +11,12 @@ export function integerToDecimal(value: BigInt, decimals: BigInt = BigInt.fromI3
   return value.toBigDecimal().div(denom.toBigDecimal());
 }
 
-export function createNewUser(address: Address): User {
+export function createNewUser(address: Address, platform: Platform): User {
   let user = new User(address.toHexString());
-  let platform = Platform.load(ZERO_ADDRESS);
-  if (platform === null) {
-    platform = createNewPlatform();
-  }
   user.operations = ZERO_BIG_INT;
   user.earned = ZERO_BIG_DECIMAL;
   platform.users = platform.users.plus(BigInt.fromI32(1));
```

Comment:
can we just increment `platform.users += 1` in the outer function that's calling this?

Reply:
That makes way more sense